lib/bundle/cask_dumper.rb

```diff
@@ -13,15 +13,33 @@ def casks
 
       require "cask/caskroom"
 
-      @casks ||= Cask::Caskroom.casks.map(&:to_s)
-      @casks.map { |cask| cask.chomp " (!)" }
-            .uniq
+      @casks ||= Cask::Caskroom.casks.map(&:to_s).uniq
+    end
+
+    def casks_config
+      return {} unless Bundle.cask_installed?
+
+      @casks_config ||= Hash[Cask::Caskroom.casks.map(&:to_s).zip(
+        Cask::Caskroom.casks.map do |cask|
+          if cask.config.nil? || cask.config.explicit.blank?
+            {}
```

Comment:
```suggestion
          next {} if cask.config.blank? || cask.config.explicit.blank?        
```

which allows you to remove the `else`